galley/pkg/metadata/kube/types.go

```diff
@@ -729,6 +729,21 @@ func init() {
 
 	versions = append(versions, "v1alpha3")
 
+	b.Add(schema.ResourceSpec{
```

Comment:
Remove this ... you don't need this type.

Reply:
@nmittler the Collection [here](https://github.com/istio/istio/blob/d9b0cdf720e58c1bc7bef2cab18f0ed2a15863bc/pilot/pkg/model/config.go#L388) is coming from `istio.io/istio/galley/pkg/metadata`. We can potentially remove that type and have the collection as a string representation e.g:
`istio/networking/v1alpha3/synthetic/serviceentries`